core/domain/skill_domain.py

```diff
@@ -676,6 +676,8 @@ def to_dict(self):
             'language_code': self.language_code,
             'version': self.version,
             'misconception_count': self.misconception_count,
-            'skill_model_created_on': self.skill_model_created_on,
-            'skill_model_last_updated': self.skill_model_last_updated
+            'created_on': utils.get_time_in_millisecs(
```

Comment:
This to_dict() wasn't used anywhere else, and when I just called it on it's own in the controllers, it threw an error that datetime cannot be copied this way (something along those lines). So, when converting to a dict, only this gave a valid dict. Also, the time is not lost, as it's just converted to a different format.
When I checked collections, in it, all the fields were manually copied in controllers, instead of using to_dict() and this conversion was applied in the controller.

Reply:
Sorry, I'm not commenting on the use of utils.get_time_in_millisecs. I'm commenting on the change of name from skill_model_created_on to created_on.

Reply:
Oh, sorry, yeah, will change the name.

Reply:
Done.